auth/auth.go

```diff
@@ -5,13 +5,32 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/url"
-	"strings"
 
 	"github.com/G-Node/gin-cli/util"
 	"github.com/G-Node/gin-cli/web"
 	"github.com/G-Node/gin-core/gin"
+	"net/http"
```

Comment:
This doesn't seem right. `net/http` related things should be handled in the lower level subpackage `web`. More on this in the general comment.